api/application.go

```diff
@@ -856,10 +1054,13 @@ type Application struct {
 	Comments        string      `json:"comments"`
 	Identities      []Ref       `json:"identities"`
 	Tags            []TagRef    `json:"tags"`
-	BusinessService *Ref        `json:"businessService"`
+	BusinessService *Ref        `json:"businessService" yaml:"businessService"`
```

Comment:
why is the yaml tag needed.  IIRC, it would default to the same camel cased.

Reply:
I'm not sure what lead to that happening. Fixing.